renderer/src/pages/settings/Settings.tsx

```diff
@@ -1,18 +1,40 @@
+import { useEffect, useState } from 'react'
+import {
+  checkForUpdates,
+  exportSeedPhrase,
+  isOpenAtLogin,
+  saveModuleLogsAs,
+  toggleOpenAtLogin
+} from 'src/lib/station-config'
+
 const Settings = () => {
+  const [isOpenAtLoginChecked, setIsOpenAtLoginChecked] = useState(true)
+
+  const updateIsOpenAtLogin = async () => setIsOpenAtLoginChecked(await isOpenAtLogin())
+
+  async function handleClick () {
+    toggleOpenAtLogin()
+    updateIsOpenAtLogin()
+  }
```

Comment:
```suggestion
  async function handleClick () {
    toggleOpenAtLogin()
    setIsOpenAtLoginChecked(!isOpenAtLoginChecked)
  }
```

should be sufficient

Reply:
That would toggle the checkbox even if for some reason the operation fails. Should we consider that possibility? If it's not an issue, I'll make the change

Reply:
This has never failed yet (I believe), the current version in tray.js also assumes it has worked

Reply:
Ok, commited 👍